kubetest2-gke/deployer/up.go

```diff
@@ -129,7 +129,10 @@ func (d *Deployer) isRetryableError(err error) bool {
 }
 
 func (d *Deployer) CreateCluster(project string, cluster cluster, subNetworkArgs []string, locationArg string) error {
-	privateClusterArgs := privateClusterArgs(d.Projects, d.Network, d.PrivateClusterAccessLevel, d.PrivateClusterMasterIPRanges, cluster)
+	privateClusterArgs := []string{}
```

Comment:
Here is the fix.

Reply:
nit: Can you break it up in 2 commits.
first which reverts the revert and a second which makes the fixes would be easier to track.

Reply:
Sure. DOne.